solutions/usaco-916.mdx

```diff
@@ -66,6 +66,39 @@ public class Revegetation {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+a, b, g = [0] * 151, [0] * 151, [0] * 101
```

Comment:
i think you know by now

Reply:
alright

Reply:
i meant the magic numbers, not the one-line assignments lol 
sorry for the miscommunication

Reply:
oh oops

Reply:
changed